saleor/graphql/order/types.py

```diff
@@ -654,6 +654,13 @@ class OrderLine(ModelObjectType[models.OrderLine]):
     total_price = graphene.Field(
         TaxedMoney, description="Price of the order line.", required=True
     )
+    undiscounted_total_price = graphene.Field(
+        TaxedMoney,
+        description=(
+            "Price of the order line without applied and order line discount."
```

Comment:
Something is wrong with the sentence.

Reply:
Right, misspelling. Thanks for catching.

Reply:
Perhaps an alternative  "Price without applying an order line discount"